orchagent/portsorch.cpp

```diff
@@ -3431,6 +3462,14 @@ bool PortsOrch::removeBridgePort(Port &port)
         return false;
     }
 
+    //Flush the FDB entires corresponding to the port
+    if (!flushFDBEntries(port.m_bridge_port_id))
+    {	    
+        SWSS_LOG_ERROR("Failed to flush FDB entries for the port %s",
+                port.m_alias.c_str());
```

Comment:
remove 'the'.
why it is different from the bridge_port_id you displayed before?
are you referring to the interface? what is port from user perspective?

Reply:
Removed 'the' word. Yes, I am referring to the interface.